astropy/units/photometric.py

```diff
@@ -45,7 +45,7 @@ def zero_point_flux(flux0):
 
     Parameters
     ----------
-    flux0 : u.Quantity
+    flux0 : :class:`~astropy.units.Quantity`
```

Comment:
Most other places Quantity is without `:class:`

Reply:
✅